metadata-models/src/main/pegasus/com/linkedin/schema/ForeignKeyConstraint.pdl

```diff
@@ -0,0 +1,37 @@
+namespace com.linkedin.schema
+import com.linkedin.common.Urn
+
+/**
+ * Description of a foreign key constraint in a schema.
+ */
+record ForeignKeyConstraint {
+  /**
+   * Name of the constraint, likely provided from the source
+   */
+  name: string
+
+  /**
+   * Fields the constraint maps to on the foreign dataset
```

Comment:
maybe a note that these are expected to be schema field urns... and what that structure looks like?

Reply:
that is already annotated in the entityTypes property below :)